core/src/main/java/org/infinispan/factories/KnownComponentNames.java

```diff
@@ -70,7 +68,7 @@
       DEFAULT_QUEUE_SIZE.put(STATE_TRANSFER_EXECUTOR, 0);
       DEFAULT_QUEUE_SIZE.put(ASYNC_OPERATIONS_EXECUTOR, 1_000);
 
-      DEFAULT_THREAD_PRIORITY.put(ASYNC_NOTIFICATION_EXECUTOR, Thread.MIN_PRIORITY);
+      DEFAULT_THREAD_PRIORITY.put(ASYNC_NOTIFICATION_EXECUTOR, Thread.NORM_PRIORITY);
```

Comment:
:)

Reply:
To be honest with you I personally think we should remove this option and just have all threads as NORM_PRIORITY always.

Reply:
Keep that in mind for your threads reorg PR :)